README.md

```diff
@@ -1 +0,0 @@
-docs/README.md
```

Comment:
Please remove this from your PR and don't modify ReadMe

Reply:
Done